decred/decred/dcr/dcrdata.py

```diff
@@ -780,7 +850,7 @@ def blockForTx(self, txid):
             return self.blockHeader(hexFromHash(bHash))
         except database.NoValue:
             # If the blockhash is not in the database, get it from dcrdata
-            decodedTx = self.dcrdata.tx(txid)
+            decodedTx = self.decodedTx(txid)
```

Comment:
Looks like this is the only place `decodedTx` is used, and the function itself is just `self.dcrdata.tx(txid)`. Instead of using `decodedTx`, let's just delete `decodedTx`.